### PR TITLE
Save default branch in user preferences

### PR DESCRIPTION
**Pull request by kuzmaka**

### CODE REVIEW

**Review by leomarlo — DISMISSED** [review]
Looks ready to go

**Comment by kuzmaka** [comment]
This one was approved once already, however approval was reset because I've merged development branch into this one to resolve merge conflict. Code diff is just the same actually.

**Review by leomarlo — APPROVED** [review]

**Pull request merged**
